patches/gnmi_set.patch

```diff
@@ -52,7 +52,16 @@
  		pbPath, err := xpath.ToGNMIPath(pathValuePair[0])
  		if err != nil {
  			log.Exitf("error in parsing xpath %q to gnmi path", pathValuePair[0])
-@@ -144,8 +166,10 @@
+ 		}
+ 		var pbVal *pb.TypedValue
```

Comment:
>TypedValue [](http://example.com/codeflow?start=17&length=10)

Why we need this patch?

Reply:
This patch introduced jwt token, and also modified path schema.

Reply:
Do we use `#` in path?

Reply:
> Do we use `#` in path?

For example, update path is /a/b/c:xyz, and then /a/b/c is path, and xyz is path value.
If the first char of path value is '@', value is a file name.
If the first char is '#', value is nil.
It's possible to use special character in path, but I don't think we use '#' as the first char in path.